WordPress/Classes/ViewRelated/Post/Revisions/SelectedRevisionLoadedProtocol.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+
+protocol SelectedRevisionLoadedProtocol {
```

Comment:
I like this protocol, but I'd rename it (much like `selectedRevisionLoaded`). Something like `RevisionLoader`, or `RevisionLoadingObserver`? I dunno exactly 😜